packages/api/prisma/schema.prisma

```diff
@@ -8,9 +8,31 @@ generator client {
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
-  githubId Int    @unique
-  fullName String
-  email    String
-  image    String
+  id              Int     @id @default(autoincrement())
+  githubId        Int     @unique
+  fullName        String?
+  email           String?
+  image           String
+  town            String?
+  gender          String?
+  birthYear       Int?
+  educationStatus String?
```

Comment:
💭 As I'm looking at the [task description](https://github.com/CodersCrew/coderscamp/issues/67#issue-947771156) only `sex` and `city` properties should be parts of the user model. After a short consideration, I think you can move even them to the `Survey` model